std/uni.d

```diff
@@ -2170,20 +2170,23 @@ public struct InversionList(SP=GcPolicy)
 
     /**
         Get range that spans all of the $(CODEPOINT) intervals in this $(LREF InversionList).
+    */
+    @property auto byInterval() scope
+    {
+        // TODO: change this to data[] once the -dip1000 errors have been fixed
+        // see e.g. https://github.com/dlang/phobos/pull/6638
+        import std.array : array;
+        return Intervals!(typeof(data.array))(data.array);
```

Comment:
Looks like this part of `-dip1000` is still under work, so I presume it's best to do the ugly allocation for now, s.t. Walter can continue to work on DIP1000 and this stays @safe.